selfservice/flow/settings/handler_test.go

```diff
@@ -402,6 +402,31 @@ func TestHandler(t *testing.T) {
 				assert.Equal(t, "You must restart the flow because the resumable session was initiated by another person.", gjson.Get(actual, "ui.messages.0.text").String(), actual)
 			})
 		})
+
+		t.Run("description=submit - kratos session cookie", func(t *testing.T) {
+			t.Run("type=api", func(t *testing.T) {
+				user1 := testhelpers.NewHTTPClientWithArbitrarySessionToken(t, reg)
+				_, body := initFlow(t, user1, true)
+				var f kratos.SelfServiceSettingsFlow
+				require.NoError(t, json.Unmarshal(body, &f))
+
+				actual, res := testhelpers.SettingsMakeRequest(t, true, false, &f, user1, `{"method":"profile", "numby": 15}`)
+				assert.Equal(t, http.StatusOK, res.StatusCode)
+				assert.Equal(t, "Your changes have been saved!", gjson.Get(actual, "ui.messages.0.text").String(), actual)
+			})
```

Comment:
This test isn't really doing anything new, maybe remove it?

Reply:
replaced with a test for spa